cli/command/network/connect.go

```diff
@@ -56,7 +63,23 @@ func runConnect(dockerCli *command.DockerCli, options connectOptions) error {
 		},
 		Links:   options.links.GetAll(),
 		Aliases: options.aliases,
+		DriverOpts: driverOpts,
 	}
 
 	return client.NetworkConnect(context.Background(), options.network, options.container, epConfig)
 }
+
+func convertDriverOpt(opts []string) (map[string]string, error) {
+	driverOpt := make(map[string]string)
+	for _, opt := range opts {
+		parts := strings.SplitN(opt, "=", 2)
+		if len(parts) != 2 {
+			err := fmt.Errorf("invalid key value pair format in driver options")
+			return nil, err
```

Comment:
Perhaps use `errors.New("invalid key/value pair format in driver options")`, instead of `fmt.Errorf()`